### ISSUE TITLE
Config file disappears on boot

### ISSUE THREAD

**Issue by xxxxme** [kitsunyan/foxy-boot#13]
The config file suggested in the description of your magisk module, /system/.magisk/img/.core/post-fs-data.d/foxy-boot.sh disappears on reboot so the appearance of the bit animation doesn't change. The .core directory doesn't exist until I create it, and it and its contents are what's deleted.

**Comment by kitsunyan** [comment]
Yes, that's because the directories were changed after some Magisk updates. Use `/data/adb/service.d` directly instead.

**Comment by Primut** [comment]
Perhaps an update to the readme is in order?
